crates/loro/src/lib.rs

```diff
@@ -1351,7 +1351,7 @@ impl LoroTree {
     }
 
     /// Return all children of the target node.
-    pub fn children(&self, parent: Option<TreeID>) -> Vec<TreeID> {
+    pub fn children(&self, parent: Option<TreeID>) -> Option<Vec<TreeID>> {
```

Comment:
Why Option? When is it None?

Reply:
when the parent does not exist or has been deleted

Reply:
Need to describe the behavior in the docs